tests/GoogleAuthenticatorTest.php

```diff
@@ -34,9 +34,11 @@ public function testGenerateSecret()
     }
 
     /**
+     * @group legacy
+     * @expectedDeprecation Passing anything other than null or a DateTimeInterface to $time is deprecated as of 2.0 and will not be possible as of 3.0.
```

Comment:
If doctrine annotations encounter an annotation that is not listed as ignored, it will crash on autoloading, this one does not (luckily), it simply ignores it. As it doesn't catch deprecations, all passes.